osbs/build/build_request.py

```diff
@@ -732,6 +737,11 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
+        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        # What about forked repos?
```

Comment:
Want to update this comment to describe the scheme we talked about in the original PR? i.e. in the future we could take spec.source.git.uri into account as well as labels, and so allow more than one build config with the same set of labels.